src/main/java/org/broadinstitute/hellbender/tools/copynumber/CallCopyRatioSegments.java

```diff
@@ -61,15 +59,46 @@
     )
     private File outFile;
 
+    @Argument(
+            doc = "Threshold on non-log2 copy ratio used for determining copy-neutral segments.  " +
+                    "If non-log2 copy ratio is within 1 +/- this threshold, a segment is considered copy-neutral.",
+            fullName = NEUTRAL_SEGMENT_COPY_RATIO_THRESHOLD_LONG_NAME,
+            shortName = NEUTRAL_SEGMENT_COPY_RATIO_THRESHOLD_SHORT_NAME,
+            optional = true
+    )
+    private double neutralSegmentCopyRatioThreshold = 0.1;
+
+    @Argument(
+            doc = "Threshold on z-score of non-log2 copy ratio used for determining outlier copy-neutral segments.  " +
+                    "If non-log2 copy ratio z-score is above this threshold for a copy-neutral segment, " +
+                    "it is considered an outlier and not used in the calculation of the length-weighted standard deviation " +
```

Comment:
I think it should say 'length-weighted mean and standard deviation used for calling the segment.'